lib/response.js

```diff
@@ -1052,6 +1052,32 @@ function sendfile(res, file, options, callback) {
 }
 
 /**
+ * Run every tracers set at the application level for given events
+ * and arguments. To set a tracer on the application use the `instrument`
+ * method.
+ *
+ * @param {function} event The event to trace.
+ * @return {ServerResponse} for chaining
+ * @public
+ */
+
+res.trace = function(event) {
+  var args = [];
+  switch (arguments.length) {
+    case 0:
+      throw new Error('No event defined!');
+    case 1: break;
+    case 2:
+      args = [arguments[1]];
+      break;
+    default:
+      args = [].slice.call(arguments, 1);
+  }
+  this.app._callTracers(this, event, args);
```

Comment:
Hm, I'm not sure how to elaborate since I don't fully understand the purpose of the tracers. Can you help me understand what kinds of things people will be putting the tracers? The example provided in this PR shows only a single tracer simply doing `debug(options.res.id, options.event, options.args.join(' '));`. Is that the only thing people will do, or are they expected to add more complex tasks there?